test/action.yml

```diff
@@ -11,6 +11,9 @@ inputs:
     description: Maven settings file path
     required: true
     default: .maven/settings.xml
+  upload_reports:
```

Comment:
```suggestion
  upload_coverage_reports:
```

Reply:
Changed in https://github.com/nimblehq/mulesoft-actions/pull/3/commits/886f1e526d6df18d0388f4853f74583d57968cef